integrations/maven-bloop/src/main/scala/bloop/integrations/maven/MojoImplementation.scala

```diff
@@ -94,15 +94,28 @@ object MojoImplementation {
         emptyLauncher
       }
 
+    val scalaContext = mojo.findScalaContext()
```

Comment:
`findScalaContext` was introduced in `maven-scala-plugin:4.5.x`. The  `basic-scala` integration test below does work because maven must use `maven-scala-plugin:4.5.x` instead of `maven-scala-plugin:3.3.2` when running the `ch.epfl.scala:maven-bloop_2.13:$version:bloopInstall` task. So far so good... but I am crossing fingers that this will work in more elaborate builds out there.

Reply:
I can add a test that uses an older version to check this out. It should work, since we don't use the version defined in the pom.

Reply:
Actually, we already have a test with an older plugin version and it works without an issue, so we should be fine.